packages/js/email-editor/src/components/header/header.tsx

```diff
@@ -94,8 +94,8 @@ export function Header() {
 	const { validateContent, isInvalid } = useContentValidation();
 
 	const { dirtyEntityRecords } = useEntitiesSavedStatesIsDirty();
-	const hasNonEmailEdits = dirtyEntityRecords.some(
-		( entity ) => entity.name !== 'mailpoet_email'
+	const hasTemplatesEdits = dirtyEntityRecords.some(
+		( entity ) => entity.name === 'wp_template'
```

Comment:
We also need to check if we edited styles because they also require the Save button. Locally, when I edited the style, the editor ended up in a state that the send button was disabled, but the save button was not displayed.
![Screenshot 2025-01-29 at 14 11 36](https://github.com/user-attachments/assets/56e60121-c06e-4a86-9def-70290e733ff2)
In the future we may also add template parts for header and footer so there might be more entities to save so I think it would be safer to stick with the `hasNonEmailEdits` logic if that's possible.
What do you think?

Reply:
Nice catch. Fixed in https://github.com/mailpoet/mailpoet/pull/6077/commits/9b01b840f54d0598968dfa4723739fbb7556b109